reference/commands/new.rst

```diff
@@ -162,7 +162,94 @@ inside the generated ``game()`` function.
 Custom templates
 ----------------
 
-There's also the possibility to create your own templates by passing a path to your template directory,
-both as an absolute path, or relative to your Conan home folder. This directory should contain Jinja2 templates,
-which will produce your desired template structure. You can use custom variables that will be needed to be passed
-as ``name`` and ```version`` does, or use your custom variables.
+There's also the possibility of creating your templates. Templates in the Conan home should be 
+located in the ``templates/command/new`` folder, and each template should have a folder named like the template one. If we create the ``templates/command/new/mytemplate``
+folder, the command will be called with the following:
+
+
+.. code-block:: bash
+
+    $ conan new mytemplate
+
+
+As other files in the Conan home, you can manage these templates with ``conan config install <url>``, putting them
+in a git repo or a http server and sharing them with your team. It is also possible to use templates from 
```

Comment:
Pedantic review warning ⚠️ 


```suggestion
in a git repo or an http server and sharing them with your team. It is also possible to use templates from 
```

@franramirez688 was correct here! (But don't tell them, they might get used to it!) As an acronym, it's spelled out letter by letter, and `h` has a vowel as its first phoneme, thus needs an `an` :)

Reply:
Thats right, thanks for the correction 😄